packages/uikit/src/atoms/Button/__snapshots__/Button.test.tsx.snap

```diff
@@ -0,0 +1,73 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Buttons Link Button renders correctly 1`] = `
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
+`;
+
+exports[`Buttons Link Button renders correctly in disabled state 1`] = `
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
+`;
+
+exports[`Buttons Link Button renders correctly in loading state 1`] = `
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
+`;
+
+exports[`Buttons Primary Button renders correctly 1`] = `
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
+`;
+
+exports[`Buttons Primary Button renders correctly in disabled state 1`] = `
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
+`;
+
+exports[`Buttons Primary Button renders correctly in loading state 1`] = `
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
+`;
+
+exports[`Buttons Secondary Button renders correctly 1`] = `
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
+`;
+
+exports[`Buttons Secondary Button renders correctly in disabled state 1`] = `
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
+`;
+
+exports[`Buttons Secondary Button renders correctly in loading state 1`] = `
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
+`;
+
+exports[`Buttons Tertiary Button renders correctly 1`] = `
+<ContextConsumer>
```

Comment:
I'm not sure that these snapshots have any value as they are all producing the same snapshot metadata whatever state they are in. I think there's probably something missing here, maybe the `mountWithTheme` method is hiding something.